test/feed.test.js

```diff
@@ -180,8 +180,8 @@ describe('Post data class tests', () => {
       // Testing Posts in ElasticSearch here
       expect(elasticPosts.results).toBe(posts.length);
       // Check whether stored text in elasticSearch is same as Post description
-      expect(elasticPosts.values[0].text).toBe(articleData1.description);
-      expect(elasticPosts.values[1].text).toBe(articleData2.description);
+      expect(elasticPosts.values[0].text).toBe(undefined);
+      expect(elasticPosts.values[1].text).toBe(undefined);
```

Comment:
undefined means there's no data, just remove these two tests

Reply:
What about others? Using mocking indexer is fine?

Reply:
Can you add `expect(Object.keys(elasticPosts.values[0])).toEqual(expect.arrayContaining(['id']));` for values[0] and values[1] here instead?